_posts/2021-07-28-how-to-update-to-cis-14.adoc

```diff
@@ -0,0 +1,349 @@
+link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/modules/security/macie[`macie` module]
+in our CIS service catalog.
+
+NOTE: Manual steps required! After updating the account baseline modules as described below, make sure you perform the manual steps
+outlined in the <<macie_manual_steps>> section.
+
+To configure account baseline modules to include Amazon Macie, add the following configuration to the respective account
+baseline module configurations:
+
+.infrastructure-live/root/_global/account-baseline/terragrunt.hcl
+[source,hcl]
+----
+inputs {
+  # ... previous inputs ...
+
+  # Configures Amazon Macie
+  create_macie_bucket      = true
+  macie_bucket_name        = "<your-macie-bucket-name>"
+  macie_create_kms_key     = true
+  macie_kms_key_name       = "<your-macie-kms-key-name>"
+  macie_kms_key_users      = ["arn:aws:iam::${local.accounts[local.account_name]}:root"]
+  macie_opt_in_regions     = local.opt_in_regions
+
+  # The variable below for Amazon Macie needs to be manually maintained. Please ensure you change the defaults.
+  macie_buckets_to_analyze = {
+    "us-east-1": ["<FILL_IN_BUCKET_1_NAME>", "<FILL_IN_BUCKET_2_NAME>"],
+    "<another-region>": ["<FILL_IN_BUCKET_3_NAME>", "<FILL_IN_BUCKET_4_NAME>"]
+  }
+}
+----
+
+.infrastructure-live/security/_global/account-baseline/terragrunt.hcl
+[source,hcl]
+----
+inputs {
+  # ... previous inputs ...
+
+  # Configures Amazon Macie
+  create_macie_bucket      = true
+  macie_bucket_name        = "<your-macie-bucket-name>"
+  macie_create_kms_key     = true
+  macie_kms_key_name       = "<your-macie-kms-key-name>"
+  macie_kms_key_users      = ["arn:aws:iam::${local.accounts[local.account_name]}:root"]
+  macie_opt_in_regions     = local.opt_in_regions
+  macie_administrator_account_id = local.accounts.root
+
+  # The variable below for Amazon Macie needs to be manually maintained. Please ensure you change the defaults.
+  macie_buckets_to_analyze = {
+    "us-east-1": ["<FILL_IN_BUCKET_1_NAME>", "<FILL_IN_BUCKET_2_NAME>"],
+    "<another-region>": ["<FILL_IN_BUCKET_3_NAME>", "<FILL_IN_BUCKET_4_NAME>"]
+  }
+}
+----
+
+NOTE: All the other child accounts (logs, stage, prod, etc) need the same configuration change as the security account above. Ensure you make that change in all the child accounts.
+
+=== Step 3: Manual steps
+
+==== Enable MFA Delete (recommendation 2.1.3)
+
+Enabling MFA Delete in your bucket adds another layer of security by requiring MFA in any request to delete a version or change the versioning state of the bucket.
+
+The attribute `mfa_delete` is only used by Terraform to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#mfa_delete[reflect the current state of the bucket]. It is not possible to create a bucket if the `mfa_delete` is `true`, because it needs to be activated https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiFactorAuthenticationDelete.html[using AWS CLI or the API].
```

Comment:
NIT: Add a sentence or two of context here first... Something like:

Unfortunately, the way AWS built the MFA delete feature is currently quite hard to use. Due to AWS API limitations, Terraform can't configure MFA delete on S3 buckets; you must first do it using the `aws` CLI, and then, after that, set `mfa_delete = true` in your Terraform code to reflect the update. Moreover, to enable MFA delete, you must authenticate as the root user of the AWS account that owns the bucket, _and_ pass in a different MFA token value _for every single bucket_ where you enable MFA delete. We've tried to make it as easy as we can, but due to how AWS built this feature, it is still quite tedious.